chopper/lib/src/chain/interceptor_chain.dart

```diff
@@ -48,7 +48,12 @@ class InterceptorChain<BodyType> implements Chain<BodyType> {
     if (index + 1 < interceptors.length &&
         interceptor is! InternalInterceptor) {
       if (response == null) {
-        throw ChopperException('Response is null', request: request);
+        throw ChopperException(
+          request: request,
+          type: ChopperExceptionType.unknown,
+          error:
```

Comment:
What's the benefit of all of this? The type is `unknown` and the error is basically the same.